.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -1,5 +1,5 @@
 name: "Validate Gradle Wrapper"
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Now it's clear what you were meaning.
https://github.com/owncloud/android-library/pull/305#issuecomment-610949497

I did'm understood exactly what you were meaning

Reply:
Ok, sorry, ill try to explain myself better next time